test/debug.sh

```diff
@@ -6,7 +6,6 @@ Localnet_Blocks_Per_Epoch=$2
 Localnet_Blocks_Per_Epoch_V2=$3
 
 ./test/kill_node.sh
-rm -rf tmp_log* 2> /dev/null
```

Comment:
let's still do the rm here so we have a proper cleanup

The assumption here is that we keep the loki logs in the DB, but we don't have to keep the old txt raw log.

Reply:
hm, good point, I'll double check this.

removed this rm, because we have an explicit `make clean` command, but probably you are right, parsed logs should be already in the loki DB

Reply:
well, if we are going this way, then it will be shift from storing logs between runs to store a Loki db.

Reply:
add in the distclean the db clean up

Reply:
yes, will do the full idea with the upcoming commit :pray: